src/GraphQL.Relay/Http/Deserializer.cs

```diff
@@ -1,80 +0,0 @@
-﻿using GraphQL;
```

Comment:
Whole this code is now taken from the server project?

Reply:
I would say that it was completely unnecessary because of the server project. This project should have no serialization or deserialization code within it.